### DISCUSSION TITLE
How to disable ESC key in ListBox

### DISCUSSION THREAD

**Discussion by dimitrisnl** [adobe/react-spectrum#2268] [Q&A] [answered]
Hey there, I've been reading the [useListBox](https://react-spectrum.adobe.com/react-aria/useListBox.html) documentation, and I've noticed that by clicking `ESC` you can clear all the selections.

Is there a way to disable/override this behaviour?

**Comment by LFDanLu** [comment, marked as answer]
You can override the `onKeyDown` returned by `useListBox` so that it early returns on `Escape`:
```
  let onKeyDown = (e) => {
    if (e.key === "Escape") {
      return;
    }

    listBoxProps.onKeyDown(e);
  };
```
See https://codesandbox.io/s/trusting-perlman-p0e5p?file=/src/App.js:557-670 for a rough example.

**Comment by dimitrisnl** [comment]
Thank you @LFDanLu , really appreciate it!
